### PR TITLE
Feature/cors

### PR DESCRIPTION
**Pull request by leomrocha**
Added CORS allowed as it is an API server.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ require('dotenv').config()
 
 var createError = require('http-errors')
 var express = require('express')
-var path = require('path')
+var cors = require('cors')
+// var path = require('path')
```
Comment (EvgeniiaVak) [inline app.js]: we don't use 'path' anymore?

File: test/test-energinet-data-api-download.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```
Comment (EvgeniiaVak) [inline test/test-energinet-data-api-download.py]: I don't think any energinet stuff should be here, maybe renaming to something like test-download-from-browser-locally.py?

**Review by EvgeniiaVak — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (leomrocha) [reply app.js]: not in this file, we use it in `routes/index.js`

File: test/test-energinet-data-api-download.py (hunk 2 above)
Comment (leomrocha) [reply test/test-energinet-data-api-download.py]: You are absolutely right. Changing name.

**Review by EvgeniiaVak — APPROVED** [review]

**Pull request merged**